NIP-65: Add note about re-publishing kind 10002 (#1889)
nostr-protocol/nips: 65.md

- Send the event to the **write** relays of the author
- Send the event to all **read** relays of each tagged user

> - Ensure the author `kind:10002` event are available in all **read** relays of each tagged user

Reviewer: ```suggestion
- Send the author's `kind:10002` event to all relays the event was published to
```
